cmd_run.go

```diff
@@ -79,21 +85,21 @@ var RunCommand = &cli.Command{
 			EnvVars: []string{"TIROS_RUN_DATABASE_SSL_MODE"},
 		},
 		&cli.StringFlag{
-			Name:    "kubo-host",
-			Usage:   "port to reach the Kubo Gateway",
-			EnvVars: []string{"TIROS_RUN_KUBO_HOST"},
+			Name:    "ipfs-host",
```

Comment:
```suggestion
			Name:    "helia-host",
```

Reply:
not sure if this is per instance or the global name

Reply:
This is per instance and would point to either Kubo or Helia. That's why I renamed it. However, I can't find any usage of that config flag anyway :D